lib/db_connection/holder.ex

```diff
@@ -332,7 +334,7 @@ defmodule DBConnection.Holder do
 
     call_reason =
       if maybe_pid do
-        "Error happened when attempting to transfer to #{inspect(maybe_pid)} " <>
+        "Error happened when attempting to transfer to #{Util.inspect_pid(maybe_pid)} " <>
```

Comment:
Note that `Util.inspect_pid/1` was updated to fall back to `inspect/1` where it's not actually a pid